index.js

```diff
@@ -1983,6 +1983,22 @@
       });
 
   //. ### List
+  //# append :: a -> [a] -> [a]
+  //.
+  //. Takes a value and a list of values of the same type and appends
+  //. the value to the list.
+  //.
+  //. ```javascript
+  //. > S.append(3, [1, 2])
+  //. [1, 2, 3]
+  //. > S.append([3, 4], [[1], [2]])
+  //. [[1], [2], [3, 4]]
+  //. ```
+  S.append =
+  def('append',
+      {},
+      [a, $.Array(a), $.Array(a)],
+      function(x, xs) { return xs.concat([x]); });
```

Comment:
github seems to be drunk. This is not the correct contents of that file. The duplicate entry of `S.append` is gone. The only one is [here](https://github.com/plaid/sanctuary/pull/184/files#diff-168726dbe96b3ce427e7fedce31bb0bcR2277)